ui/js/directives/treeherder/clonejobs.js

```diff
@@ -526,29 +557,29 @@ treeherder.directive('thCloneJobs', [
 
             //Empty the job column before populating it
             jobTdEl.empty();
-
-            var jgObj, jobGroup, i;
+            var jgObj, jobGroup, i, jobTdHtml = "";
             for (i=0; i<jobGroups.length; i++) {
 
                 jgObj = jobGroups[i];
-
                 if (jgObj.symbol !== '?') {
                     // Job group detected, add job group symbols
                     jobGroups[i].grkey = jgObj.mapKey;
                     jobGroups[i].collapsed = true;
-                    jobGroup = $(jobGroupInterpolator(jobGroups[i]));
-                    jobTdEl.append(jobGroup);
-
+                    jobGroups[i].btnHTML = getJobBtnEls(jgObj);
                     if (isGroupExpanded(jgObj)) {
-                        addJobBtnEls(jgObj, jobGroup.find(".group-job-list"));
+                        jobGroups[i].btnHTML = addJobBtnEls(jgObj);
```

Comment:
I don't think this will have the effect that we want, since addJobBtnEls will call renderJobBtnEls, which seems to do a bunch of DOM manipulation. So I'd be surprised if we're getting much benefit from things as they stand.

We need to rework this function so it doesn't call any jquery selectors (`$(...)`) at all while it's working. Only string manipulations.